jetty-server/src/main/java/org/eclipse/jetty/server/HttpChannelState.java

```diff
@@ -819,11 +819,16 @@ protected void onError(Throwable th)
         {
             // If we are still async and nobody has called sendError
             if (_requestState == RequestState.ASYNC && !_sendError)
+            {
                 // Then the listeners did not invoke API methods
                 // and the container must provide a default error dispatch.
                 sendError(th);
-            else
+            }
+            // Otherwise the listeners have called AsyncContext.complete().
```

Comment:
this comment comment is outside the if, so should be phrased as a conditional, or moved inside the if body

Reply:
I'm not a big fan of comments outside `if/else` that just say what the `if/else` is doing.
I'm for removing both.